domains/boxbb.is-cool.dev.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "../schemas/domain.schema.json",
+
+  "description": "Portfolio website",
+
+  "domain": "is-cool.dev",
+  "subdomain": "boxbb",
+
+  "owner": {
+    "email": "manyu34@gmail.com"
+  },
+
+  "record": {
+    "NS": ['NS1.DYNU.COM', 'NS7.DYNU.COM', 'NS9.DYNU.COM', 'NS11.DYNU.COM'],
```

Comment:
Please provide the specific and detailed reason why you need NS records as with the readme.md guidelines.